schema/provisioner.proto

```diff
@@ -6,5 +6,5 @@ import "stake.proto";
 message Provisioner {
     bytes public_key_bls = 1;
     bytes raw_public_key_bls = 2;
-    repeated Stake stakes = 3;
```

Comment:
I would avoid changes proto definition right now
What about leaving the repeated fields and populate only the first atm?
This avoid last-minute changes on the node side, while keeping proto ready for the future